core/server/api/canary/oembed.js

```diff
@@ -1,8 +1,55 @@
 const common = require('../../lib/common');
-const {extract, hasProvider} = require('oembed-parser');
+const { extract, hasProvider } = require('oembed-parser');
 const Promise = require('bluebird');
 const request = require('../../lib/request');
 const cheerio = require('cheerio');
+const domino = require('domino');
```

Comment:
Do we need to introduce another DOM dependency here? Wondering if it's possible to use `cheerio` because we already use that in a number of other places

Reply:
Ignore me, I was thinking of `jsdom` not `cheerio`, and we've already dropped `jsdom` 🙈 `domino` seems like a fine choice here 👍

Reply:
We'll actually not need even `domino` I think as `metascraper` won't need the coversion